CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
 # 1.6.0-PRE1
 
-- References librdkafka 1.6.0. Refer to the [release notes](https://github.com/edenhill/librdkafka/releases/tag/v1.6.0) for more information. Headline features:
+- References librdkafka 1.6.0-PRE1. Refer to the [release notes](https://github.com/edenhill/librdkafka/releases/tag/v1.6.0) for more information. Headline features:
   - KIP-429: Incremental rebalancing.
   - KIP-480: Sticky partitioner.
+- KIP-22: Support for custom partitioners.
+- Librdkafka is no longer prevented from being loaded on Linux or MacOS when targeting `net452` (allows Confluent.Kafka to be used with Mono on these platforms).
+- Exceptions thrown by `Producer.Produce` now include an inner exception with additional context on the error ([joostas](https://github.com/joostas)).
```

Comment:
A small code blob showing how to get at the inner exception would be fantastic!

Reply:
it's standard practice and the norm everywhere else except this method, so i think it would be out of place. it's automatically covered by our examples since they print exceptions with ToString, which will print the name of the InnerException by default.